src/cms/static/js/region-form-icon.js

```diff
@@ -0,0 +1,20 @@
+u('#icon').on('change', updateIconDisplay);
+
+/**
+ * listen for user input (selection in file browser)
+ * for uploading region icons
+ * updates the icon preview and checkbox for clearing
+ */
+function updateIconDisplay() {
+    const curFiles = u('#icon').first().files;
+    if(curFiles.length === 0) {
+        u('#icon_status').text("No icon selected");
+        u('#clear_icon').addClass("hidden");
+        u('#clear_icon').removeClass("block");
```

Comment:
```suggestion
```
If the div is hidden, you don't need to remove the block class as well. It's not shown anyway.